site/docs-md/apis/splash-screen/index.md

```diff
@@ -44,6 +44,18 @@ If your app needs longer than 3 seconds to load, configure the default duration
 }
 ```
 
+If you want to be sure the splash never hides before the app is fully loaded, set `launchAutoHide` to `false` in your `capacitor.config.json`:
+
+```json
+{
+  "plugins": {
+    "SplashScreen": {
+      "launchAutoHide": false
```

Comment:
Can you also add it to the "Configuration" section? (just a few lines below)

Reply:
Sorry, missed it – now updated.